src/nifgen/metadata/functions_addon.py

```diff
@@ -139,7 +139,10 @@
     'WaitUntilDone':                                { 'parameters': { 1: { 'python_api_converter_name': 'convert_timedelta_to_milliseconds',
                                                                            'type_in_documentation': 'float in seconds or datetime.timedelta', }, }, },
     'InitializeWithChannels':                       { 'parameters': { 3: { 'python_api_converter_name': 'convert_init_with_options_dictionary', 
-                                                                           'type_in_documentation': 'dict', }, }, },
+                                                                           'type_in_documentation': 'dict', }, 
+                                                                      1: { 'name': 'channels',  # need to set the name so it doesn't look like a repeated capability to the codegenerator. Could also use 'is_repeated_capability': False
```

Comment:
Setting `is_repeated_capability = False` seems more correct. I can see us adding `channels` to the list of strings that we infer to be `is_repeated_capability = True`

Reply:
Changed